src/components/Table/Table.jsx

```diff
@@ -41,6 +41,13 @@ const getTable = (data, columns, sortState, sortField, sortClick) => (
                 overflow: `${column.field === 'kills' ? 'visible' : null}`,
                 color: column.color,
               };
+              if (row.value === null && column.field !== 'name') {
```

Comment:
for reference:
https://github.com/odota/ui/blob/master/src/actions/player/playerRecordsActions.js

You can just add a .filter() onto the .map and remove anything where value is null or undefined